File: projetArmeeBack/src/main/java/projetArmee/projetArmeeBack/entities/ArmeeAir.java
package projetArmee.projetArmeeBack.entities;

import javax.persistence.Column;
import javax.persistence.DiscriminatorValue;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@DiscriminatorValue("Air")
public class ArmeeAir extends Armee {

	public ArmeeAir() {
	}

	public ArmeeAir(Integer moral, Pays pays) {
		super(100, moral, pays);
		
	}

	

	
}

File: projetArmeeAngular/projetArmee/src/app/models/armeeAir.ts
import { Pays } from './pays';
import { Unite } from './unite';
export class Armee {
  constructor(
    public type?: string,
    public pv?: number,
    public moral?: number,
    public pays?: Pays,
    public listeUnites?: Set<Unite>,
    public id?: number
  ) {}
}
